docs/usage/api/python-api.md

```diff
@@ -17,11 +17,32 @@ By default there are two JavaScript modules. More can be added with
       - The JavaScript Pyodide module.
 ```
 
+**Python Modules**
+
+````{eval-rst}
+.. list-table::
+
+   *  - :py:mod:`pyodide.code`
+      - Utilities for evaluating Python and JavaScript code.
+   *  - :py:mod:`pyodide.console`
+      - Similar to the Python builtin `code` module but handles top level await. Used
+        for implementing the Pyodide console.
+   *  - :py:mod:`pyodide.ffi`
+      - The :any:`JsProxy` class and utilities to help interact with JavaScript code.
+   *  - :py:mod:`pyodide.http`
+      - Defines :any:`pyfetch` and other functions for making network requests.
+   *  - :py:mod:`pyodide.webloop`
+      - The Pyodide event loop implementation. This is automatically configured
+        correctly for most use cases it is unlikely you will need it outside of niche
+        use cases.
+
 ```{eval-rst}
-.. currentmodule:: pyodide
+.. toctree::
+   :hidden:
 
-.. automodule:: pyodide
-   :members:
-   :autosummary:
-   :autosummary-no-nesting:
-```
+   python-api/code.md
+   python-api/console.md
+   python-api/ffi.md
+   python-api/http.md
+   python-api/webloop.md
+````
```

Comment:
This toctree is not correctly parsed.

https://pyodide--2790.org.readthedocs.build/en/2790/usage/api/python-api.html